Source/Core/Common/ChunkFile.h

```diff
@@ -235,7 +235,7 @@ class PointerWrap
 
 		while (true)
 		{
-			u8 shouldExist = (list_cur ? 1 : 0);
+			u8 shouldExist = !!list_cur;
```

Comment:
1. true/false are [always](https://stackoverflow.com/a/4276239/272237) 0/1. You're probably thinking of [the opposite](https://msdn.microsoft.com/en-us/library/b6801kcy.aspx).
2. I thought `!!` is a well-known pattern? It makes it easier to read imho...